.ado/azure-pipelines.yml

```diff
@@ -23,17 +23,21 @@ jobs:
       - script: |
           yarn checkchange
         displayName: "check change"
-
       - script: |
           yarn check-for-changed-files
         displayName: "verify API and Ensure Changed Files"
+      # Creates a variable that determines whether the previous tasks succeeded.
+      - task: PowerShell@2
+        inputs:
+          targetType: "inline"
+          script: |
+            Write-Host "##vso[task.setvariable variable=task.A.status]Success"
```

Comment:
can you change the variable name? task.A.status doesn't mean much. Also if you could elaborate here, what this is doing. Not quite sure I understand what's being accomplished.

Reply:
If it's used within e2e-testing-windows exclusively maybe it makes sense as the first task in that template?

Reply:
Will change name and move to e2e-testing-windows. Good idea!

Reply:
Updated in newest commit and added a more in-depth comment. It's a little confusing and hard to put down in words why exactly this is needed, if you think it needs more clarification, I will update it accordingly.